versioned_docs/version-4.0.0/tributech_node/Webhook_integration.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Webhooks Integration
+sidebar_position: 2
+---
+
+## General 
+We provide the possibility within our Tributech Node for external services to receive a wide variety of notifications about internal events via [Webhooks](https://en.wikipedia.org/wiki/Webhook). A webhook is an HTTP-based callback function that enables lightweight, event-driven communication between a Tributech Node and a client. The only requirement for a client is to provide a HTTP POST endpoint that returns Status code 200 if a event was successfully received or any other status code when the event has not been processed successfully. When a client reports a problem with an event Demeter will respond according to our [Error Handling](#error-handling) logic.
```

Comment:
>The only requirement for a client is to provide a HTTP POST endpoint that returns Status code 200 if a event was successfully received or any other status code when the event has not been processed successfully

I think we should rephrase it , its not about processing rather than having a success in receiving the webhook itself. Because demeter will kick off the error handling (= redilevery) but can not change the metadata information of the event itself. So if an event had an error or the client had an error with processing the Webhook Event itself it should also sent back a 200.